readme.md

```diff
@@ -1,44 +1,95 @@
 
-## Generating Test Files
-Test files will fail at least one (and sometimes more than one) edit. These files will be used as a double check for the implementation of the HMDA edits for the 2018 HMDA Platform. 
+-`geography_data.yaml` holds the filepaths for geographic dependency data such as the tract to CBSA file. The configuration includes the column names for the CBSA file and the columns used for file creation. The file also contains a path to a list of zip codes and a dictionary of FIPS state codes. 
 
-Creating test files is a two step process. The generation of edit test files requires a clean data file to be present. A clean file is created when [generate_clean_files](https://github.com/cfpb/hmda-test-files/blob/master/python/generate_clean_files.py) is run. Running [generate test files](https://github.com/cfpb/hmda-test-files/blob/master/python/generate_error_files.py) modifies the data from the clean file and saves it with the name of the edit check that it will fail. If edit files already exist with the same names as those produced by this script, they will be overwrriten.
+-`clean_file_config.yaml` holds values for the test institution, file length, and file name for generating a clean HMDA submission file for testing. Other variables, such as data ranges can also be set in this file. 
 
-To generate test files:
-- `python generate_clean_files.py`: running [this script](https://github.com/cfpb/hmda-test-files/blob/master/python/generate_clean_files.py) creates clean synthetic data file. This data file passes HMDA edit rules. The data ranges used in file creation are configured by making changes to the [configuration file](https://github.com/cfpb/hmda-test-files/blob/master/python/config.yaml) which is written in YAML.
-- `python generate_error_files`: running [this script](https://github.com/cfpb/hmda-test-files/blob/master/python/generate_error_files.py) produces a file that will fail conditions for each edit (other edits may also fail). These files are written to the appropriate directory based on edit type, such as /edits_files/syntax/s300.txt. The file name indicates which edit the file is designed to test. If an edit has multiple conditions, a file will be made for each condition in the format /edits_files/syntax/s301_1.txt.
+### Generating Clean Files and Edit Test Files. 
+The function method `create_files(kind)` outputs a clean test file or a set of edit test files, with an input indicating the kind of file to be created. Currently, the two inputs are `"clean_file"` which creates a clean file and `"error_files"` which produces files that will fail conditions for each edit. Edit test files are written to the directory indicated in `test_filepaths.yaml` based on edit type, such as /edits_files/test_files/Bank1/syntax/s300.txt. The file name indicates which edit the file is designed to test. If an edit has multiple conditions, a file will be made for each condition in the format /edits_files/test_files/Bank1/syntax/s301_1.txt.
```

Comment:
Is this `two inputs` referring to `kind`?

Reply:
Hopefully clarified this in a new section referenced in the next commit!